testcase failed in node v0.11.13

``` bash
$ npm test

> gulp-unzip@0.1.0 test /private/tmp/gulp-unzip
> mocha

  ․․․․․

  1 passing (8s)
  4 failing

  1) gulp-unzip basic:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/private/tmp/gulp-unzip/node_modules/mocha/lib/runnable.js:139:19)
      at Timer.listOnTimeout (timers.js:133:15)

  2) gulp-unzip large_file:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/private/tmp/gulp-unzip/node_modules/mocha/lib/runnable.js:139:19)
      at Timer.listOnTimeout (timers.js:133:15)

  3) gulp-unzip filter option false filter:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/private/tmp/gulp-unzip/node_modules/mocha/lib/runnable.js:139:19)
      at Timer.listOnTimeout (timers.js:133:15)

  4) gulp-unzip filter option only extract css with filter:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/private/tmp/gulp-unzip/node_modules/mocha/lib/runnable.js:139:19)
      at Timer.listOnTimeout (timers.js:133:15)

npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   "author": "suisho",
   "license": "BSD-2-Clause",
   "dependencies": {
-    "unzip": "~0.1.9",
+    "unzip": "^0.1.11",
     "gulp-util": "~2.2.14",
     "through2": "~0.4.1",
     "defaults": "^1.0.0"

